src/GithubIssues.py

```diff
@@ -624,7 +624,7 @@ def get_issue_ages(self, issue_type, month_end_date):
                                  parse_dates=['opened_date'],
                                  params=(self.repository_id, issue_type, month_end_date, month_end_date))
             self.status = 0
-            return result.applymap(lambda x: (pd.to_datetime(month_end_date) - x).days)
+            return result.applymap(lambda x: (pd.to_datetime(month_end_date, utc=True) - x).days)
```

Comment:
I wondered the same thing. I decided I would only add `utc=True` for this single instance, since that was all that was needed to make it work for my data. I'm not sure if this is needed elsewhere. As far as I can tell, the other places where datetime subtraction is performed may not have the issue at all (because both inputs would have timezones _or_ neither would have timezones).